src/frontend/qt_sdl/CMakeLists.txt

```diff
@@ -70,21 +69,41 @@ find_package(Threads REQUIRED)
 find_package(PkgConfig REQUIRED)
 find_package(Iconv REQUIRED)
 pkg_check_modules(SDL2 REQUIRED sdl2)
-pkg_check_modules(SLIRP REQUIRED slirp)
+pkg_check_modules(SLIRP slirp)
 pkg_check_modules(LIBARCHIVE REQUIRED libarchive)
 add_compile_definitions(ARCHIVE_SUPPORT_ENABLED)
 
+if (SLIRP_FOUND)
+    option(ENABLE_LIBSLIRP "Enable use of libslirp for network sockets" ON)
+else()
+    option(ENABLE_LIBSLIRP "Enable use of libslirp for network sockets" OFF)
+endif()
+
 if (WIN32 AND (CMAKE_BUILD_TYPE STREQUAL Release))
     add_executable(melonDS WIN32 ${SOURCES_QT_SDL})
 else()
     add_executable(melonDS ${SOURCES_QT_SDL})
 endif()
 
+if (ENABLE_LIBSLIRP)
+	target_compile_definitions(melonDS PRIVATE -DHAVE_LIBSLIRP)
+    target_sources(melonDS PRIVATE LAN_Socket.cpp)
```

Comment:
I think you mixed up tabs and spaces here.

Reply:
fark

We really need an editorconfig file.